modules/persistent-storage-csi-drivers-supported.adoc

```diff
@@ -18,8 +18,8 @@ The following table describes the CSI drivers that are installed with {product-t
 ifndef::openshift-dedicated,openshift-rosa[]
 |AliCloud Disk | ✅ | - | ✅ | -
 endif::openshift-dedicated,openshift-rosa[]
-|AWS EBS | ✅ | - | ✅| -
-|AWS EFS | - | - | -| -
+|Amazon EBS | ✅ | - | ✅
+|Amazon EFS | - | - | -
```

Comment:
Revoked on Line 21 and Line 22.

Reply:
Updated, Amazon Elastic Block Store (EBS) and Amazon Elastic File Storage (EFS) as this is the first service name instance here.